libs/common-services/src/main/java/uk/gov/di/ipv/core/library/service/IpvSessionService.java

```diff
@@ -35,6 +35,11 @@ public class IpvSessionService {
     private static final String START_STATE = "START";
     private static final String ERROR_STATE = "ERROR";
 
+    // Max sleeping time will be roughly WAIT_TIME * 2 ^ (MAX_ATTEMPTS - 1), plus execution time
+    // AWS say 'Consistency across all copies of data is usually reached within a second'
+    private static final int MAX_ATTEMPTS = 5;
+    private static final int WAIT_TIME_MILLIS = 50;
```

Comment:
More precisely, this config will spend up to 750ms sleeping (plus the time spent executing the DynamoDB requests), and the old config would have spent up to 630ms sleeping.